.github/workflows/pipeline.yml

```
@@ -173,13 +173,20 @@ jobs:
                   node-version: 20.x
             - name: Install pnpm modules
               run: pnpm install --frozen-lockfile
-            - name: Replace placeholder with instrumentation key
+            - name: Replace placeholder with instrumentation key (MCP)
               uses: jacobtomlinson/gha-find-replace@v3
               with:
                   include: packages/fiori-mcp-server/src/telemetry/index.ts
                   find: 'ApplicationInsightsInstrumentationKeyPLACEH0LDER'
                   replace: ${{ secrets.INSTRUMENTATION_KEY }}
                   regex: false
+            - name: Replace placeholder with instrumentation key (SAP Systems Extension)
+              uses: jacobtomlinson/gha-find-replace@v3
+              with:
+                  include: packages/sap-systems-ext/src/utils/telemetryHelper.ts
+                  find: 'ApplicationInsightsInstrumentationKeyPLACEH0LDER'
+                  replace: ${{ secrets.INSTRUMENTATION_KEY }}
+                  regex: false
```

```suggestion
            - name: Replace placeholder with instrumentation key (SAP Systems Extension)
              uses: jacobtomlinson/gha-find-replace@v3
              with:
                  include: 'packages/sap-systems-ext/src/utils/telemetryHelper.ts|packages/fiori-mcp-server/src/telemetry/index.ts'
                  find: 'ApplicationInsightsInstrumentationKeyPLACEH0LDER'
                  replace: ${{ secrets.INSTRUMENTATION_KEY }}
                  regex: true
```
could be avoid adding a step for each package released with telemetry by using regex include or other option?